app/services/waste_carriers_engine/registration_completion_service.rb

```diff
@@ -73,13 +73,14 @@ def delete_transient_registration
       transient_registration.delete
     end
 
+    # Note that we will only send emails here if the registration has pending convictions or pending payments.
+    # In the case when the registration can be completed, the registration activation email is sent from
+    # the RegistrationActivationService.
     def send_confirmation_email
       if registration.unpaid_balance?
         send_pending_payment_email
-        # TODO: Add email for pending convictions
-        # Note that we will only send emails here if the registration has pending convictions or pending payments.
-        # In the case when the registration can be completed, the registration activation email is sent from
-        # the RegistrationActivationService.
+      elsif registration.conviction_check_required?
```

Comment:
I would keep the comment, just not the first `TODO` line.

Reply:
It's still there! Just moved it up a few lines.